numba/cuda/stubs.py

```diff
@@ -1,17 +1,19 @@
 """
 This scripts specifies all PTX special objects.
 """
-import operator
-import numpy
+import functools
 import llvmlite.llvmpy.core as lc
+import numpy
```

Comment:
I know this is just forwarding the original code, but for consistency in the code base this should be `import numpy as np`

Reply:
it turned out not to be required at all so I removed it.